lib/remap-functions/src/lib.rs

```diff
@@ -195,3 +195,106 @@ pub use truncate::Truncate;
 pub use upcase::Upcase;
 #[cfg(feature = "uuid_v4")]
 pub use uuid_v4::UuidV4;
+
+pub fn all() -> Vec<Box<dyn remap::Function>> {
+    let mut functions: Vec<Box<dyn remap::Function>> = vec![];
+
+    #[cfg(feature = "md5")]
+    functions.push(Box::new(Md5));
```

Comment:
This function only runs at startup, so it doesn't matter for performance, but I wonder if Rust might emit less code with an array construct like:
```suggestion
    let functions: [Box<dyn remap::Function>] = [

        #[cfg(feature = "md5")]
        Box::new(Md5),
        …
    ];
    functions.iter().collect()
```
In any case, this could probably be done within the `vec![]` macro instead of with explicit pushes.

Reply:
Good catch, will fix 👍